core/local_vector.h

```diff
@@ -69,15 +70,15 @@ class LocalVector {
 		if (!std::is_trivially_constructible<T>::value && !force_trivial) {
 			memnew_placement(&data[count++], T(p_elem));
```

Comment:
I think this could be `T(std::move(p_elem))`, but I see I have missed this in my original PR myself :D